warehouse/query-core/src/main/java/datawave/core/iterators/DatawaveFieldIndexRangeIteratorJexl.java

```diff
@@ -107,12 +107,25 @@ public SortedKeyValueIterator<Key,Value> deepCopy(IteratorEnvironment env) {
         return new DatawaveFieldIndexRangeIteratorJexl(this, env);
     }
 
+    @Override
+    public String toStringNoQueryId() {
+        return toStringImpl(false);
```

Comment:
Why override if identical to parent method

Reply:
Fixed